client/me/help/help-results/index.jsx

```diff
@@ -16,6 +17,8 @@ export default function HelpResults( {
 		return null;
 	}
 
+	onLinksDisplay( helpLinks );
```

Comment:
This will be called with every render. I think it should in inside an effect. Maybe can send the event from here without propagating it upwards?

Reply:
Thanks @alshakero for the review! Tried a new approach - what do you think?